Dockerfile.7.0

```diff
@@ -22,10 +22,13 @@ RUN yum install -y centos-release-scl-rh && \
     yum clean all -y
 
 # Install wordpress and backup the base image S2I scripts
-RUN cd /tmp && curl -o wordpress.tar.gz -SL https://wordpress.org/wordpress-${WORDPRESS_VERSION}.tar.gz \
+RUN echo "Install wordpress and backup the base image S2I scripts" \
+    && cd /tmp && curl -o wordpress.tar.gz -SL https://wordpress.org/wordpress-${WORDPRESS_VERSION}.tar.gz \
     && mkdir -p /opt/app-root/wordpress \
     && echo "$WORDPRESS_SHA1 *wordpress.tar.gz" | sha1sum -c - \
     && tar -xzf wordpress.tar.gz --strip-components=1 -C /opt/app-root/wordpress \
+	&& rm -rf /opt/app-root/wordpress/wp-content/plugins/akismet \
+	&& rm -f /opt/app-root/wordpress/wp-content/plugins/hello.php \
```

Comment:
what does hello.php do? I am just curious to know

Reply:
https://wordpress.org/plugins/hello-dolly/
"When activated you will randomly see a lyric from Hello, Dolly in the upper right of your admin screen on every page."
The plugin was not activated and used in the Bonnier sites so we can clean up